crates/symbolicator/src/endpoints/applecrashreport.rs

```diff
@@ -46,8 +46,13 @@ async fn handle_apple_crash_report_request(
     let report = report.ok_or_else(|| error::ErrorBadRequest("missing apple crash report"))?;
 
     let symbolication = state.symbolication();
-    let request_id =
-        symbolication.process_apple_crash_report(params.scope, report, sources, options);
+    let request_id = symbolication
+        .process_apple_crash_report(params.scope, report, sources, options)
+        .ok_or_else(|| {
+            error::ErrorServiceUnavailable(
+                "Symbolicator is currently processing the maximum number of requests.",
```

Comment:
I assume this is the error message you're referring to that's a bit of a WIP: I'm thinking we could maybe say something like "Cannot process this request: symbolicator has reached the maximum number of concurrent requests ({what the maximum is})".

Is this string being returned as-is to sentry? Who exactly sees this error string?

Reply:
The WIP message I was referring to was literally "TODO", but I'm not super happy with this one either. Now that I look at them I see that other error messages in these files are very terse, and IIUC nobody should ever actually see these.

Reply:
👍 the new message looks a lot better and is in line with the existing ones in the project